### PR TITLE
Fix pl-dropdown to allow unescaped values

### PR DESCRIPTION
**Pull request by mwest1066**

### CODE REVIEW

File: elements/pl-dropdown/pl-dropdown.mustache [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <div id="pl-dropdown-{{uuid}}" class="d-inline-block form-group form-inline">
         <select name="{{answers-name}}" class="form-control">
             {{#options}}
-                <option value="{{value}}" {{#selected}}selected{{/selected}}>{{value}}</option>
+                <option value="{{value}}" {{#selected}}selected{{/selected}}>{{{value}}}</option>
```
Comment (echuber2) [inline elements/pl-dropdown/pl-dropdown.mustache:15]: Does the value attribute need triple braces also?

Comment (jonatanschroeder) [reply elements/pl-dropdown/pl-dropdown.mustache:15]: That's not even enough. There are some problems when the value submitted in the form includes the &. Reviewing now.

Comment (jonatanschroeder) [reply elements/pl-dropdown/pl-dropdown.mustache:15]: It works properly without unescaping value. I added the change in submitted and correct panels as well.

**Review by jonatanschroeder — APPROVED** [review]

**Comment by mwest1066** [comment]
@jonatanschroeder Good call on the submitted and correct answers!

**Pull request merged**